Cargo.toml

```diff
@@ -14,6 +14,6 @@ blst = "0.3.6"
 rand = "0.8.0"
 thiserror = "1.0.30"
 sha2 = "0.9.8"
-hex = "0.4.3"
```

Comment:
lets leave this in :)

Reply:
whoops will do, thought it was only being used for my old `hex::decode` stuff.